### DISCUSSION TITLE
Subscript in raw text

### DISCUSSION THREAD

**Discussion by trya2l** [typst/typst#3804] [Q&A] [answered]
Hi,

As I have read in the documentation, `raw`
> Displays the text verbatim and in a monospace font.

However, I would like to know if there is a 'non verbatim' alternative to `raw`

Especially, I would like to print something like `S#sub[1]` in a raw like mode.

Do you have some recommendations, because I have tried using text, but using `Linux Libertine Mono` does not bring the same output as `raw` ?

Thanks :)

**Comment by frozolotl** [comment]
You might want to consider using a package for writing pseudocode, like [lovelace](https://typst.app/universe/package/lovelace).
Alternatives: [algo](https://typst.app/universe/package/algo) and [algorithmic](https://typst.app/universe/package/algorithmic).

**Reply by trya2l** [reply]
Thanks for the answer, but this doesn't seem to fit my needs. The output is not as expected

![screen](https://github.com/typst/typst/assets/79646756/64121319-28b4-4c41-a44e-3f0088aafb1f)

**Comment by 8LWXpg** [comment, marked as answer]
Two solutions I have:

**Using Unicode character [subscript one](https://www.compart.com/en/unicode/U+2081)**

````typ
```
S₁
```
````

![image](https://github.com/typst/typst/assets/105704427/f3888e05-1bae-4ff7-9a1b-1381ad922fe2)

**Setting font to `DejaVu Sans Mono` with `.8em` font size**
```typ
#text(font: "DejaVu Sans Mono", size: .8em)[S#sub[1]]
```
![image](https://github.com/typst/typst/assets/105704427/7bc94f30-be3c-4492-87bb-966e8d2b8892)

**Reply by trya2l** [reply]
Oh great, thanks for this one. I think this might do  for what I need.
